### PR TITLE
Release notes for helm chart 1.8.0

### PR DESCRIPTION
**Pull request by jedcunningham**
Time to release 1.8.0!

### CODE REVIEW

File: chart/RELEASE_NOTES.rst [hunk 1]
```diff
@@ -23,15 +23,115 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
...
+
+Default git-sync image is updated to ``3.6.3`` (#27848)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The default git-sync image that is used with the Chart is now ``3.6.3``, previously it was ``3.4.0``.
+
+Default redis image is updated to ``7-bullseye`` (#27443)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The default git-sync image that is used with the Chart is now ``7-bullseye``, previously it was ``6-bullseye``.
```
Comment (potiuk) [inline chart/RELEASE_NOTES.rst]:
```suggestion
The default redis image that is used with the Chart is now ``7-bullseye``, previously it was ``6-bullseye``.
```

Comment (jedcunningham) [reply chart/RELEASE_NOTES.rst]: 🤦‍♂️

**Review by potiuk — APPROVED** [review]
Small nit.

**Comment by potiuk** [comment]
Small comment for the spellcheck problems - the spellcheck problems is why in the Provider changelog I surround all commit messages with double-backquotes - those are not spell-checked and even if they are a little "geeky" in the changelog, it's the only sane way I could find generating 70+ changelogs with 100s of commits works :D

Might be not that important in case of the Helm chart, but wanted to share a friendly release mangement tip.

**Comment by jedcunningham** [comment]
I get why you take that approach with the providers, that's a lot of volume.

I'm trying to make the release notes more user-centric ultimately, so having those failures is actually helpful - I can go back and reword them to be even more user-centric 🎉

File: chart/RELEASE_NOTES.rst [hunk 2]
```diff
@@ -23,15 +23,115 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
 
 .. towncrier release notes start
 
+Airflow Helm Chart 1.8.0 (2023-02-02)
```
Comment (jedcunningham) [inline chart/RELEASE_NOTES.rst:26]:
```suggestion
Airflow Helm Chart 1.8.0 (2023-02-03)
```

**Review by kaxil — APPROVED** [review]

**Review by ephraimbuddy — APPROVED** [review]

File: chart/Chart.yaml [hunk 3]
```diff
@@ -67,187 +67,143 @@ annotations:
...
+    - description: Add support for ``webserverConfigConfigMapName``
+      kind: changed
       links:
-      - name: '#22815'
-        url: https://github.com/apache/airflow/pull/22815
-    - description: Add support for KEDA HPA config to Helm chart
-      kind: added
+      - name: '#27419'
+        url: https://github.com/apache/airflow/pull/27419
+    - description: Enhance chart to allow over-riding command-line args to statsd exporter
```
Comment (ephraimbuddy) [inline chart/Chart.yaml]:
```suggestion
    - description: Enhance chart to allow overriding command-line args to statsd exporter
```

File: chart/RELEASE_NOTES.rst [hunk 4]
```diff
@@ -23,15 +23,116 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
...
+^^^^^^^^^^^^
+
+- Add annotations on deployments (#28688)
+- Add global volume & volumeMounts to the chart (#27781)
+
+Improvements
+^^^^^^^^^^^^
+
+- Add support for ``webserverConfigConfigMapName`` (#27419)
+- Enhance chart to allow over-riding command-line args to statsd exporter (#28041)
```
Comment (ephraimbuddy) [inline chart/RELEASE_NOTES.rst]:
```suggestion
- Enhance chart to allow overriding command-line args to statsd exporter (#28041)
```

**Review by ephraimbuddy — APPROVED** [review]

**Pull request merged**